cli/sdk.go

```diff
@@ -58,6 +62,17 @@ func (c *CLI) InitializeManagerClient(cmd *cobra.Command) error {
 	return nil
 }
 
+func (c *CLI) RunMeasurement(filePathToMeasure string, igvmBinaryPath string) error {
+	if c.measurement == nil {
```

Comment:
if there is no other place measurement is set then we do not need this conditional statement